### DISCUSSION TITLE
Assembly of chloroplast genome results in no circular structure

### DISCUSSION THREAD

**Discussion by wu19971108** [Kinggerm/GetOrganelle#146] [Q&A]
The result is only one file,embplant_pt.K105.scaffolds.graph1.1.path_sequence.fasta.
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8251169/get_org.log.txt)
![3VTQ01APHOC87VRB }6V$DK](https://user-images.githubusercontent.com/101625368/158330923-fb5c6344-269f-46de-b823-e20c303dd203.png)

**Comment by Kinggerm** [comment]
Sorry for the delay. Please have the `Depth` box checked so that we can see the coverages of contigs.

The good news is that this graph seems to be plastome-sufficient, where we may likely to achieve the complete plastome.

**Comment by wu19971108** [comment]
&nbsp; Thank you for your reply. My picture is as follows


---Original---
From: "Jianjun ***@***.***&gt;
Date: Wed, Mar 23, 2022 10:11 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Kinggerm/GetOrganelle] Assembly of chloroplast genome results inno circular structure (Discussion #146)




 
Sorry for the delay. Please have the Depth box checked so that we can see the coverages of contigs.
 
The good news is that this graph seems to be plastome-sufficient, where we may likely to achieve the complete plastome.

**Reply by Kinggerm** [reply]
I can't see pictures available
